datatree/ops.py

```diff
@@ -213,16 +213,16 @@ def method_name(self, *args, **kwargs):
         if wrap_func is map_over_subtree:
             # Add a paragraph to the method's docstring explaining how it's been mapped
             orig_method_docstring = orig_method.__doc__
-            if orig_method_docstring is not None:
-                if "\n" in orig_method_docstring:
-                    new_method_docstring = orig_method_docstring.replace(
-                        "\n", _MAPPED_DOCSTRING_ADDENDUM, 1
-                    )
-                else:
-                    new_method_docstring = (
-                        orig_method_docstring + f"\n\n{_MAPPED_DOCSTRING_ADDENDUM}"
-                    )
-                setattr(target_cls_dict[method_name], "__doc__", new_method_docstring)
+            # if orig_method_docstring is not None:
+            #     if "\n" in orig_method_docstring:
+            #         new_method_docstring = orig_method_docstring.replace(
+            #             "\n", _MAPPED_DOCSTRING_ADDENDUM, 1
+            #         )
+            #     else:
+            #         new_method_docstring = (
+            #             orig_method_docstring + f"\n\n{_MAPPED_DOCSTRING_ADDENDUM}"
+            #         )
+            setattr(target_cls_dict[method_name], "__doc__", orig_method_docstring)
```

Comment:
@TomNicholas - I got the build to work but it required rolling back this bit of code. Something in here was making sphinx very unhappy. It may be worth copying the approach Dask is taking (https://github.com/dask/dask/blob/153353d50e2c4ffc58380eef5a61baf208b2e69b/dask/utils.py#L637-L671)